python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -318,3 +313,67 @@ def quantize(self, pl_model,  # remove the type requirement for type checking
                     quantized_onnx_model)
         else:
             raise NotImplementedError("Backend {} is not implemented.".format(backend))
+
+    @staticmethod
+    def trace(model: nn.Module, input_sample=None, accelerator=None):
+        """
+        Trace a pytorch model and convert it into an accelerated module for inference.
+        For example, this function returns a PytorchOpenVINOModel when accelerator=='opevino'.
+
+        :param model: An torch.nn.Module model, including pl.LightningModule.
+        :param input_sample: A set of inputs for trace, defaults to None if you have trace before or
+                             model is a LightningModule with any dataloader attached.
+        :param accelerator: The accelerator to use, defaults to None meaning staying in Pytorch
+                            backend. Only 'openvino' is supported for now.
```

Comment:
I wonder if we need to provide a `trace_args` for our users to tune the onnx tracing setting. This may enlarge our availability.

Reply:
Let's include this for onnx later. For openvino, no further args are used for now.